windows/searchlibrary/0f7750bd-1576-481a-b60d-b382bc9e70fd.query

```diff
@@ -6,10 +6,7 @@
 tag=$WINDOWS_LOGON winlog TimeCreated EventID == 4624 Computer SubjectUserName SubjectDomainName TargetUserName TargetDomainName LogonType LogonProcessName WorkstationName as TargetInfo IpAddress
 | lookup -s -r @groups [Computer SubjectUserName SubjectDomainName TargetUserName TargetDomainName LogonType] [Comp SUN SDN TUN TDN LT]
 | lookup -r windows_login_types LogonType logon_type name as LogonName
-| dns -reverse IpAddress Host
-| printf -e SourceInfo "%v (%v)" Host IpAddress
-| printf -e LogonTypeInfo "%v (%v)" LogonName LogonType
 | printf -e SubjectUser "%v\\%v" SubjectDomainName SubjectUserName

```

Comment:
you deleted the LogonTypeInfo, this query will fail to parse

Reply:
Yeah caught that...already fixing